.gitignore

```diff
@@ -1,5 +1,7 @@
+.puli/
 build/
 vendor/
 composer.lock
 phpspec.yml
 phpunit.xml
+puli.json
```

Comment:
Puli works differently than Composer. There's no need for a lock file in Puli.

You are right that in this case, a puli.json is not needed (but neither is a lock file).